module/localisation/FieldLocalisation/src/FieldLocalisation.cpp

```diff
@@ -122,13 +128,37 @@ namespace module::localisation {
                     filter.time(dt);
 
                     auto field(std::make_unique<Field>());
-                    field->Hfw        = compute_Hfw(filter.get_state());
+                    field->Hfw = compute_Hfw(filter.get_state());
```

Comment:
Let's add a configuration option where you can select between using filter or ground truth

Reply:
Something like
```
if(cfg.use_ground_truth){
        field->Hfw = raw_sensors.localisation_ground_truth.Hfw;
} else {
        field->Hfw = compute_Hfw(filter.get_state());
}
```